### PR TITLE
master lpd 42601

### PR DESCRIPTION
**Pull request by kyle-miho**
cc: @drewbrokke @nicolas-sss

### CODE REVIEW

File: modules/util/source-formatter/src/test/java/com/liferay/source/formatter/processor/UpgradeCatchAllCheckTest.java [hunk 1]
```diff
@@ -70,12 +72,32 @@ public static Iterable<Object[]> data() throws Exception {
...
+						});
 				}
 			}
 		}
 
-		return objectsArray;
+		Set<Object[]> objectSet = new HashSet<>();
+
+		for (Map.Entry<String, Set<String>> stringSetEntry :
+				setMap.entrySet()) {
```
Comment (drewbrokke) [inline modules/util/source-formatter/src/test/java/com/liferay/source/formatter/processor/UpgradeCatchAllCheckTest.java:93]: Check other uses of iterating `map.getEntrySet`. Double-check the variable names for the `stringSetEntry` variable and the `setMap` variable here, whether or not they should be "generic" names (e.g. `map`, `entry`) or "purpose-built" names (e.g. `issueKeyFileTypesMap`,  `issueKeyFileTypesEntry`) or some combination of the two.

In any case, I think we should change the current variable names.

**Review by drewbrokke — CHANGES_REQUESTED** [review]

**Comment by kyle-miho** [comment]
ci:test:sf

**Comment by kyle-miho** [comment]
ci:test:hotel

**Review by drewbrokke — APPROVED** [review]

**Comment by drewbrokke** [comment]
Looks good, thanks @kyle-miho!

**Comment by kyle-miho** [comment]
https://github.com/brianchandotcom/liferay-portal/pull/156792

**Pull request closed without merging**